tpl/template_funcs.go

```diff
@@ -2186,6 +2186,7 @@ func initFuncMap() {
 		"substr":       substr,
 		"title":        title,
 		"time":         asTime,
+		"now":          func() time.Time { return time.Now() },
```

Comment:
I know this fits fine next to time, but the rest of them are alphabetic ordered.